template/node/Dockerfile

```diff
@@ -33,7 +33,7 @@ COPY function/*.json ./
 RUN npm i --production || :
 
 # Copy in additional function files and folders
-COPY --chown=app:app function/ .
+COPY function/ .
```

Comment:
There is a "chown app:app -R /home/app" at line 42, and the path of function is /home/app/function.
So the owner of /home/app/function/* will change after line 42.

Reply:
I'm asking you to do this from experience with our build tooling (https://github.com/moby/buildkit) - it requires the chmod, please restore it to the previous variation by checking git history to see when I switched to `--chown`.